tedana/combine.py

```diff
@@ -158,23 +158,32 @@ def make_optcom(data, tes, mask, t2s=None, combmode='t2s', verbose=True):
         LGR.warning("Argument 't2s' is not required if 'combmode' is 'paid'. "
                     "'t2s' array will not be used.")
 
-    data = data[mask, :, :]  # mask out empty voxels/samples
-    tes = np.array(tes)[np.newaxis, ...]  # (1 x E) array_like
-
     if combmode == 'paid':
-        LGR.info('Optimally combining data with parallel-acquired inhomogeneity '
-                 'desensitized (PAID) method')
-        combined = _combine_paid(data, tes)
+        LGR.info('Optimally combining data with parallel-acquired '
+                 'inhomogeneity desensitized (PAID) method')
     else:
         if t2s.ndim == 1:
             msg = 'Optimally combining data with voxel-wise T2* estimates'
         else:
             msg = ('Optimally combining data with voxel- and volume-wise T2* '
                    'estimates')
-        t2s = t2s[mask, ..., np.newaxis]  # mask out empty voxels/samples
-
         LGR.info(msg)
-        combined = _combine_t2s(data, tes, t2s)
+
+    mask = adaptive_mask >= 3
+    data = data[mask, :, :]  # mask out empty voxels/samples
```

Comment:
I know we've had some confusion on this in the past, so I'm wondering if we can head it off with the code comment :sweat_smile: 

By `empty` you mean not having signal across at least three echos, right ? Maybe we could call them `unstable` instead of `empty` ? Not a huge point, just want to make sure this is clear. Maybe we could actually update this in the docstring for `make_optcom` itself ?